### PR TITLE
Added per vendor price in cart.

### PR DESCRIPTION
**Pull request by Shikhar1903**

### CODE REVIEW

File: app/src/main/java/com/dvm/appd/oasis/dbg/notification/FirebaseMessagingService.kt [hunk 1]
```diff
@@ -104,10 +105,10 @@ class FirebaseMessagingService : FirebaseMessagingService() {
         }
             .subscribe({
                 Log.d("Notification", "Succesfully added data to table")
-                /*val bundle = Bundle()
+                val bundle = Bundle()
                 bundle.putString(FirebaseAnalytics.Param.ITEM_ID, notificatoin.id)
                 bundle.putString(FirebaseAnalytics.Param.ITEM_NAME, notificatoin.title)
-                bundle.putString(FirebaseAnalytics.Param.CONTENT_TYPE, "Book")*/
+                bundle.putString(FirebaseAnalytics.Param.CONTENT_TYPE, "Book")
```
Comment (PrarabdhGarg) [inline app/src/main/java/com/dvm/appd/oasis/dbg/notification/FirebaseMessagingService.kt]: Please comment this back

**Comment by IshitaAg** [comment]
Remove hardcoded width and height in adapter_cart_items.xml

File: app/src/main/java/com/dvm/appd/oasis/dbg/wallet/views/adapters/CartAdapter.kt [hunk 2]
```diff
@@ -52,6 +59,7 @@ class CartAdapter(private val listener: CartChildAdapter.OnButtonClicked): Recyc
 
         val vendor: TextView = view.vendor
         val items: RecyclerView = view.itemsRecycler
+        val vendor_Price: TextView=view.per_vendor_price
```
Comment (PrarabdhGarg) [inline app/src/main/java/com/dvm/appd/oasis/dbg/wallet/views/adapters/CartAdapter.kt]: Add a space before and after the equal to sign

File: app/src/main/java/com/dvm/appd/oasis/dbg/wallet/views/fragments/OrdersFragment.kt [hunk 3]
```diff
@@ -54,13 +54,16 @@ class OrdersFragment : Fragment(), OrdersAdapter.OrderCardClick, CartChildAdapte
...
+
             view.noCart.isVisible = it.isEmpty()
             (view.cartItemRecycler.adapter as CartAdapter).cartItems = it
             (view.cartItemRecycler.adapter as CartAdapter).price = it.sumBy { it1 -> it1.second.sumBy {it2 ->  it2.quantity * it2.currentPrice }}
             (view.cartItemRecycler.adapter as CartAdapter).notifyDataSetChanged()
 
             if (it.sumBy { it1 -> it1.second.sumBy {it2 ->  it2.quantity * it2.currentPrice  }} != 0) {
                 view.order.isVisible = true
+
+
```
Comment (PrarabdhGarg) [inline app/src/main/java/com/dvm/appd/oasis/dbg/wallet/views/fragments/OrdersFragment.kt]: Remove the extra lines

File: app/src/main/res/layout/adapter_cart_item.xml [hunk 4]
```diff
@@ -23,18 +23,16 @@
 
     <TextView
             android:id="@+id/vendor"
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
+            android:layout_width="282dp"
+            android:layout_height="31dp"
```
Comment (PrarabdhGarg) [inline app/src/main/res/layout/adapter_cart_item.xml]: Remove hard-coded width and height

**Review by PrarabdhGarg — APPROVED** [review]

**Pull request merged**